docs/participate/03_treasury_proposal.md

```diff
@@ -11,6 +11,46 @@ The first step in applying for a Treasury grant is either to join the community
 This will help you get valuable community feedback throughout the process.
 It also gives Council members an open and transparent way to measure community sentiment.
 
+When submitting for discussion, it is best practice to include a proposal document.
+[A reference document](https://docs.google.com/document/d/1nz5GNjTL5inaI5s9djIki7DUp5zyPJV9dEI9K06kJ58/edit) can be found in the KILT Protocol discussions this can be used as a template for future proposals.
+
+## DID Sign
+
+The proposed document MUST be DID signed.
+This requires a DID that MUST be associated with the proposal.
+Therefore, it requires the proposer to have a DID.
+The signature provides integrity and accountability for the submitter, which can give more confidence to the community and council.
+
+The DID Signature should be done via [DIDSign](https://didsign.io/) a dApp which is built on KILT.
+The DIDSign doesn't have a database, nor store any data by the user.
+The following guide provides a [how to create a signature using a DID](https://kilt-protocol.org/files/How-to-Guide-DIDsign.pdf) with DIDSign.
+
+Once the document has been signed and a link can be added to the proposal so others can verify.
+This MUST include the document and signature.
+The method of storing the document and the signature can vary, however, we recommend using IPFS or an equivelant such as Crust.
```

Comment:
We don't really store the document, as the document itself can be downloaded from Google, most likely. If you want to highlight that other providers can be used, it should be explained. The way it is currently written does not make it very easy to read, at least for me.